.github/workflows/auto_build.yml

```
@@ -0,0 +1,233 @@
+
+        # Kiểm tra xem có Gradle Wrapper không
+        if [ -f "gradlew" ]; then
+          echo "HAS_WRAPPER=true" >> $GITHUB_OUTPUT
+          echo "Gradle Wrapper detected"
+        else
+          echo "HAS_WRAPPER=false" >> $GITHUB_OUTPUT
+          echo "No Gradle Wrapper found"
+        fi
+
+    - name: Setup JDK 17
+      uses: actions/setup-java@v3
```

actionlint (1.7.7)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)